validation/src/test/scala/hmda/validation/rules/lar/macro/Q007Spec.scala

```diff
@@ -12,23 +12,25 @@ class Q007Spec extends MacroSpec {
 
   val testLars = lar100ListGen.sample.getOrElse(Nil)
   val sampleSize = testLars.size
+  def relevantLar(lar: LoanApplicationRegister) = lar.copy(actionTakenType = 4)
+  def irrelevantLar(lar: LoanApplicationRegister) = lar.copy(actionTakenType = 2)
```

Comment:
(As we discussed -- I'm just commenting here for transparency) Let's switch the definitions of `relevantLar` and `irrelevantLar`. For the purpose of Macro edits, we're defining "relevant" as the LARs that meet the criteria in the edit (aka LARs that are counted in this edit). This will make the logic clearer for defining `numOfRelevantLars` too